tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -119,3 +126,77 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+External URLs Should Not Be Broken
+    [Documentation]     Go through a QuickStart and checks the status of all the external links
+    ${quick_start_steps}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    URLs HTTP Status Code Should Be Equal To    ${element_list}
+
+    FOR    ${quick_start_step}    IN     @{quick_start_steps}
+            Open QuickStart Step  ${quick_start_step}
+            ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+            URLs HTTP Status Code Should Be Equal To    ${element_list}
+            ${Doc_Text}     Get Text  //*[@class="pf-c-drawer__body pf-m-no-padding pfext-quick-start-panel-content__body"]
+            ${Doc_links}     Get Regexp Matches   ${Doc_Text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
+            IF  ${Doc_links}
+                Split Doc and Valiadte URLS     ${Doc_links}
+            END
+    END
+
+Verify Links Are Not Broken For Each QuickStart
+    [Documentation]     Clicks on al the quick start and verify links
+    [Arguments]    ${quickStartElements}
+    ${quickStartCount}=   Get Length           ${quickStartElements}
+    ${TitleElements}=     Get WebElements      //div[@class="pf-c-card__title odh-card__doc-title"]
+    FOR    ${counter}    IN RANGE     ${quickStartCount}
+        Log To Console    \n++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ${Title}=   Get Text          ${TitleElements[${counter}]}
+        ${Title}=   Split To Lines    ${Title}
+        Log To Console                ${Title[${0}]}
+        Click Element                 ${quickStartElements[${counter}]}
+        Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
+        External URLs Should Not Be Broken
+    END
+
+Get QuickStart Items
+        Wait Until Resource Page Is Loaded
+        ${quickStartElements}=     Wait for QuickStart to Load
+        [Return]    ${quickStartElements}
+
+
+Open QuickStart Step
+    [Documentation]   Click next if next not found cick tab to find buttion
+    [Arguments]  ${quick_start_step}
+    ${status}   Run Keyword And Return Status   Click Element  ${quick_start_step}
+    IF  ${status} == False
+        Click Button    Next
+    END
+
+    FOR    ${counter}    IN RANGE    5
+        Press Keys    NONE    TAB
+    END
+
+
+Split Doc and Valiadte URLS
```

Comment:
hm not sure about the name "Split Doc..". Maybe, sth clearer would be `Filter And Validate Links Extracted From Text` or Just `Validate Links Extracted From Text`